domain-management/src/main/java/org/jboss/as/domain/management/connections/ldap/LdapConnectionAddHandler.java

```diff
@@ -125,7 +127,17 @@ static Config updateRuntime(final OperationContext context, final ModelNode mode
             handlesReferralsForSet = Collections.emptySet();
         }
 
-        return connectionManagerService.setConfiguration(initialContextFactory, url, searchDn, searchCredential, referralHandling, handlesReferralsForSet);
+        final boolean alwaysSendClientCert;
+        if (model.hasDefined(ALWAYS_SEND_CLIENT_CERT.getName())) {
+            /* If always-send-client-cert is defined in the management model, take it */
+            alwaysSendClientCert = ALWAYS_SEND_CLIENT_CERT.resolveModelAttribute(context, model).asBoolean();
+        } else {
+            /* Otherwise, use the system property if set, or false as default */
+            alwaysSendClientCert = Boolean.valueOf(WildFlySecurityManager
+                    .getPropertyPrivileged("jboss.as.management.outbound.ldap.alwaysSendClientCert", "false"));
```

Comment:
why introducing new system property? 
You have defined configuration property, as such I don't see any good reason to have this extra property here.

Also, if new property is really *needed* it should be added as default value to the attribute and resolve that instead of all this checking here.